internal/config/config.go

```diff
@@ -5,9 +5,35 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/ghodss/yaml"
 	"github.com/ilyakaznacheev/cleanenv"
 )
 
+var (
+	defaultConfigYAML = []byte(`
+log:
+  level: info
+  format: json
+admin:
+  listeners:
+  - address: ":3000"
+    protocol: http
+database:
+  query_timeout: 5s
+metrics:
+  enabled: true
+disable_anonymous_reports: false
+`)
+	defaultConfig Config
+)
+
+func init() {
+	err := yaml.Unmarshal(defaultConfigYAML, &defaultConfig)
+	if err != nil {
+		panic(fmt.Errorf("failed to decode default config: %v", err))
+	}
+}
```

Comment:
@hbagdi in order to properly implement this, I need to make updates to some (all?) of the types by including `omitempty` so that zero-values don't overwrite config defaults. Is that the intended expected behavior?

Reply:
Are we adding this default config only for testing purposes?

Reply:
Nevermind, this was added mistakenly during a rebase. I've resolved.